content/blog/2023-04-13.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Version 1.0.0 released!"
+date: 2023-04-13T13:48:11+02:00
+description: "Version 1.0.0 released!"
+draft: false
+---
+
+In this 1.0.0 release, we focused on supporting [Apache Camel](https://camel.apache.org/) DSLs, such as [Camel Route in YAML DSL](https://camel.apache.org/camel-k/1.11.x/languages/yaml.html), [Kamelet](https://camel.apache.org/camel-k/1.12.x/kamelets/kamelets-user.html), [KameletBinding](https://camel.apache.org/camel-k/1.12.x/kamelets/kamelets-user.html#kamelets-usage-binding) and [Integration](https://camel.apache.org/camel-k/1.12.x/running/running.html#no-cli-integration).
```

Comment:
I would put this paragraph on the second place. First we say this is the major release. Then we say on what we focused.

Reply:
I updated the order as follows:

1. This marks the first major release for Kaoto...
2. Kaoto was built to be extended,...
3. In this 1.0.0 release, we focused on supporting Camel

I hope that this is better now :smiley: